sktime/benchmarking/_lib_mini_kotsu/run.py

```diff
@@ -6,6 +6,7 @@
 from typing import List, Optional, Union
 
 import pandas as pd
+from tqdm import tqdm
```

Comment:
Is `tqdm` part of our core dependencies, or implied by those? If not, this is going to fail.

Reply:
I'm afraid tqdm is an independent package

Reply:
I'm not soubting that, but it could be implied. For example, `statsforecast` (part of `forecasting` subset) will imply `tqdm` and so on.

As far as I know none of the core dependencies, e.g. `pandas`, `numpy`, `scikit-learn` etc. will imply `tqdm`, so this line will fail CI. What you should do is move this import to inside your functions/methods and use `tqdm` only if it present. There is no need to make progress bars mandatory, especially if someone uses it in non-interactive setup.

Reply:
@yarnabrina
What dependency do u suggest using for adding the progress bar